code/src/index.css

```diff
@@ -11,3 +14,171 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+p {
+  color: #fff;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* Navbar */ 
+.nav-bar {
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+}
+
+.nav-bar a {
+  color: #fff;
+  font-size: 24px;
+  padding: 10px;
+}
+
+.nav-section {
```

Comment:
Maybe move the nav-section parts above the .nav-bar parts sense it is the parent, just to make it even easier to read?